web/prisma/migrations/20240307185725_backfill_score_source/migration.sql

```diff
@@ -0,0 +1,32 @@
+-- Set default values conditionally with explicit casting using a batched update
+DO
+$$
+DECLARE 
+   r record;
+   rows_processed int;
+BEGIN 
+   LOOP
+      rows_processed := 0;
+      FOR r in 
+         SELECT "name", "id"
+         FROM "scores"
+         WHERE "source" IS NULL
+         LIMIT 1000
+         FOR UPDATE 
```

Comment:
I think we are good with this one as our environment is not highly concurrent. We mostly deal with historic data. For update gets a lock on the row which might timeout when doing that for 1k or more entries.

Reply:
thanks